### ISSUE TITLE
Android blank

### ISSUE THREAD

**Issue by TrustTheBoy** [henninghall/react-native-date-picker#188]
**Describe the bug**
Android blank

**Expected behavior**

**To Reproduce**
![image](https://user-images.githubusercontent.com/20201085/81663458-48b10f00-9471-11ea-9dff-4500ed3413a3.png)
```javascript
const [date, setDate] = useState(new Date())
<DatePicker
   date={date}
   onDateChange={setDate}
/>				
```

**Smartphone (please complete the following information):**
 - OS: [Android]
 - React Native version [e.g. 0.61]
 - react-native-date-picker version [e.g. 2.7.9]

**Comment by TrustTheBoy** [comment]
android studio console:
```console
 java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.String.length()' on a null object reference
    at java.text.SimpleDateFormat.validatePattern(SimpleDateFormat.java:276)
    at java.text.SimpleDateFormat.<init>(SimpleDateFormat.java:365)
    at com.henninghall.date_picker.wheels.DayWheel.getDisplayValueFormat(DayWheel.java:108)
    at com.henninghall.date_picker.wheels.DayWheel.getDisplayValue(DayWheel.java:100)
    at com.henninghall.date_picker.wheels.DayWheel.getValues(DayWheel.java:43)
    at com.henninghall.date_picker.wheels.Wheel.init(Wheel.java:94)
    at com.henninghall.date_picker.wheels.Wheel.refresh(Wheel.java:76)
    at com.henninghall.date_picker.wheelFunctions.Refresh.apply(Refresh.java:9)
    at com.henninghall.date_picker.ui.Wheels.applyOnAll(Wheels.java:77)
    at com.henninghall.date_picker.ui.UIManager.updateDisplayValues(UIManager.java:53)
    at com.henninghall.date_picker.PickerView.update(PickerView.java:60)
    at com.henninghall.date_picker.DatePickerManager.onAfterUpdateTransaction(DatePickerManager.java:63)
    at com.henninghall.date_picker.DatePickerManager.onAfterUpdateTransaction(DatePickerManager.java:25)
    at com.facebook.react.uimanager.ViewManager.updateProperties(ViewManager.java:50)
    at com.facebook.react.uimanager.NativeViewHierarchyManager.createView(NativeViewHierarchyManager.java:278)
     at com.facebook.react.uimanager.UIViewOperationQueue$CreateViewOperation.execute(UIViewOperationQueue.java:179)
     at com.facebook.react.uimanager.UIViewOperationQueue$DispatchUIFrameCallback.dispatchPendingNonBatchedOperations(UIViewOperationQueue.java:972)
     at com.facebook.react.uimanager.UIViewOperationQueue$DispatchUIFrameCallback.doFrameGuarded(UIViewOperationQueue.java:943)
     at com.facebook.react.uimanager.GuardedFrameCallback.doFrame(GuardedFrameCallback.java:28)
     at com.facebook.react.modules.core.ReactChoreographer$ReactChoreographerDispatcher.doFrame(ReactChoreographer.java:174)
     at com.facebook.react.modules.core.ChoreographerCompat$FrameCallback$1.doFrame(ChoreographerCompat.java:84)
     at android.view.Choreographer$CallbackRecord.run(Choreographer.java:856)
     at android.view.Choreographer.doCallbacks(Choreographer.java:670)
     at android.view.Choreographer.doFrame(Choreographer.java:603)
     at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:844)
     at android.os.Handler.handleCallback(Handler.java:739)
     at android.os.Handler.dispatchMessage(Handler.java:95)
     at android.os.Looper.loop(Looper.java:148)
     at android.app.ActivityThread.main(ActivityThread.java:5554)
     at java.lang.reflect.Method.invoke(Native Method)
     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:935)
     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:726)
 20404-20404/com.xxx I/TIME4A: Starting Time4A (v4.2-2018i published on 2019-01-02)
05-12 16:59:38.609 20404-20404/com.xxx I/TIME4A: Main-Thread consumed in ms: 0
  java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.String.length()' on a null object reference
      at java.text.SimpleDateFormat.validatePattern(SimpleDateFormat.java:276)
      at java.text.SimpleDateFormat.<init>(SimpleDateFormat.java:365)
      at com.henninghall.date_picker.wheels.DayWheel.getDisplayValueFormat(DayWheel.java:108)
      at com.henninghall.date_picker.wheels.DayWheel.getDisplayValue(DayWheel.java:100)
      at com.henninghall.date_picker.wheels.DayWheel.getValues(DayWheel.java:43)
      at com.henninghall.date_picker.wheels.Wheel.init(Wheel.java:94)
      at com.henninghall.date_picker.wheels.Wheel.refresh(Wheel.java:76)
      at com.henninghall.date_picker.wheelFunctions.Refresh.apply(Refresh.java:9)
      at com.henninghall.date_picker.ui.Wheels.applyOnAll(Wheels.java:77)
      at com.henninghall.date_picker.ui.UIManager.updateDisplayValues(UIManager.java:53)
      at com.henninghall.date_picker.PickerView.update(PickerView.java:60)
      at com.henninghall.date_picker.DatePickerManager.onAfterUpdateTransaction(DatePickerManager.java:63)
      at com.henninghall.date_picker.DatePickerManager.onAfterUpdateTransaction(DatePickerManager.java:25)
      at com.facebook.react.uimanager.ViewManager.updateProperties(ViewManager.java:50)
      at com.facebook.react.uimanager.NativeViewHierarchyManager.createView(NativeViewHierarchyManager.java:278)
      at com.facebook.react.uimanager.UIViewOperationQueue$CreateViewOperation.execute(UIViewOperationQueue.java:179)
      at com.facebook.react.uimanager.UIViewOperationQueue$DispatchUIFrameCallback.dispatchPendingNonBatchedOperations(UIViewOperationQueue.java:972)
      at com.facebook.react.uimanager.UIViewOperationQueue$DispatchUIFrameCallback.doFrameGuarded(UIViewOperationQueue.java:943)
      at com.facebook.react.uimanager.GuardedFrameCallback.doFrame(GuardedFrameCallback.java:28)
      at com.facebook.react.modules.core.ReactChoreographer$ReactChoreographerDispatcher.doFrame(ReactChoreographer.java:174)
      at com.facebook.react.modules.core.ChoreographerCompat$FrameCallback$1.doFrame(ChoreographerCompat.java:84)
      at android.view.Choreographer$CallbackRecord.run(Choreographer.java:856)
      at android.view.Choreographer.doCallbacks(Choreographer.java:670)
      at android.view.Choreographer.doFrame(Choreographer.java:603)
      at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:844)
      at android.os.Handler.handleCallback(Handler.java:739)
      at android.os.Handler.dispatchMessage(Handler.java:95)
      at android.os.Looper.loop(Looper.java:148)
      at android.app.ActivityThread.main(ActivityThread.java:5554)
      at java.lang.reflect.Method.invoke(Native Method)
      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:935)
      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:726)
 20404-20404/com.xx I/TIME4A: Starting Time4A (v4.2-2018i published on 2019-01-02)
 20404-20404/com.xx I/TIME4A: Main-Thread consumed in ms: 0
  java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.String.length()' on a null object reference
      at java.text.SimpleDateFormat.validatePattern(SimpleDateFormat.java:276)
     at java.text.SimpleDateFormat.<init>(SimpleDateFormat.java:365)
      at com.henninghall.date_picker.wheels.DayWheel.getDisplayValueFormat(DayWheel.java:108)
      at com.henninghall.date_picker.wheels.DayWheel.getDisplayValue(DayWheel.java:100)
      at com.henninghall.date_picker.wheels.DayWheel.getValues(DayWheel.java:43)
      at com.henninghall.date_picker.wheels.Wheel.init(Wheel.java:94)
      at com.henninghall.date_picker.wheels.Wheel.refresh(Wheel.java:76)
      at com.henninghall.date_picker.wheelFunctions.Refresh.apply(Refresh.java:9)
      at com.henninghall.date_picker.ui.Wheels.applyOnAll(Wheels.java:77)
      at com.henninghall.date_picker.ui.UIManager.updateDisplayValues(UIManager.java:53)
      at com.henninghall.date_picker.PickerView.update(PickerView.java:60)
      at com.henninghall.date_picker.DatePickerManager.onAfterUpdateTransaction(DatePickerManager.java:63)
      at com.henninghall.date_picker.DatePickerManager.onAfterUpdateTransaction(DatePickerManager.java:25)
      at com.facebook.react.uimanager.ViewManager.updateProperties(ViewManager.java:50)
      at com.facebook.react.uimanager.NativeViewHierarchyManager.createView(NativeViewHierarchyManager.java:278)
      at com.facebook.react.uimanager.UIViewOperationQueue$CreateViewOperation.execute(UIViewOperationQueue.java:179)
      at com.facebook.react.uimanager.UIViewOperationQueue$1.run(UIViewOperationQueue.java:773)
      at com.facebook.react.uimanager.UIViewOperationQueue.flushPendingBatches(UIViewOperationQueue.java:888)
      at com.facebook.react.uimanager.UIViewOperationQueue.access$2200(UIViewOperationQueue.java:42)
      at com.facebook.react.uimanager.UIViewOperationQueue$DispatchUIFrameCallback.doFrameGuarded(UIViewOperationQueue.java:948)
      at com.facebook.react.uimanager.GuardedFrameCallback.doFrame(GuardedFrameCallback.java:28)
      at com.facebook.react.modules.core.ReactChoreographer$ReactChoreographerDispatcher.doFrame(ReactChoreographer.java:174)
      at com.facebook.react.modules.core.ChoreographerCompat$FrameCallback$1.doFrame(ChoreographerCompat.java:84)
      at android.view.Choreographer$CallbackRecord.run(Choreographer.java:856)
      at android.view.Choreographer.doCallbacks(Choreographer.java:670)
      at android.view.Choreographer.doFrame(Choreographer.java:603)
      at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:844)
      at android.os.Handler.handleCallback(Handler.java:739)
      at android.os.Handler.dispatchMessage(Handler.java:95)
      at android.os.Looper.loop(Looper.java:148)
      at android.app.ActivityThread.main(ActivityThread.java:5554)
      at java.lang.reflect.Method.invoke(Native Method)
      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:935)
      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:726)
 20404-20404/com.mobileinspection I/TIME4A: Starting Time4A (v4.2-2018i published on 2019-01-02)
 20404-20404/com.mobileinspection I/TIME4A: Main-Thread consumed in ms: 0
  java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.String.length()' on a null object reference
      at java.text.SimpleDateFormat.validatePattern(SimpleDateFormat.java:276)
      at java.text.SimpleDateFormat.<init>(SimpleDateFormat.java:365)
      at com.henninghall.date_picker.wheels.DayWheel.getDisplayValueFormat(DayWheel.java:108)
      at com.henninghall.date_picker.wheels.DayWheel.getDisplayValue(DayWheel.java:100)
      at com.henninghall.date_picker.wheels.DayWheel.getValues(DayWheel.java:43)
      at com.henninghall.date_picker.wheels.Wheel.init(Wheel.java:94)
      at com.henninghall.date_picker.wheels.Wheel.refresh(Wheel.java:76)
      at com.henninghall.date_picker.wheelFunctions.Refresh.apply(Refresh.java:9)
      at com.henninghall.date_picker.ui.Wheels.applyOnAll(Wheels.java:77)
      at com.henninghall.date_picker.ui.UIManager.updateDisplayValues(UIManager.java:53)
      at com.henninghall.date_picker.PickerView.update(PickerView.java:60)
      at com.henninghall.date_picker.DatePickerManager.onAfterUpdateTransaction(DatePickerManager.java:63)
      at com.henninghall.date_picker.DatePickerManager.onAfterUpdateTransaction(DatePickerManager.java:25)
      at com.facebook.react.uimanager.ViewManager.updateProperties(ViewManager.java:50)
      at com.facebook.react.uimanager.NativeViewHierarchyManager.createView(NativeViewHierarchyManager.java:278)
      at com.facebook.react.uimanager.UIViewOperationQueue$CreateViewOperation.execute(UIViewOperationQueue.java:179)
      at com.facebook.react.uimanager.UIViewOperationQueue$1.run(UIViewOperationQueue.java:773)
      at com.facebook.react.uimanager.UIViewOperationQueue.flushPendingBatches(UIViewOperationQueue.java:888)
      at com.facebook.react.uimanager.UIViewOperationQueue.access$2200(UIViewOperationQueue.java:42)
      at com.facebook.react.uimanager.UIViewOperationQueue$DispatchUIFrameCallback.doFrameGuarded(UIViewOperationQueue.java:948)
      at com.facebook.react.uimanager.GuardedFrameCallback.doFrame(GuardedFrameCallback.java:28)
      at com.facebook.react.modules.core.ReactChoreographer$ReactChoreographerDispatcher.doFrame(ReactChoreographer.java:174)
      at com.facebook.react.modules.core.ChoreographerCompat$FrameCallback$1.doFrame(ChoreographerCompat.java:84)
      at android.view.Choreographer$CallbackRecord.run(Choreographer.java:856)
      at android.view.Choreographer.doCallbacks(Choreographer.java:670)
      at android.view.Choreographer.doFrame(Choreographer.java:603)
      at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:844)
      at android.os.Handler.handleCallback(Handler.java:739)
      at android.os.Handler.dispatchMessage(Handler.java:95)
      at android.os.Looper.loop(Looper.java:148)
      at android.app.ActivityThread.main(ActivityThread.java:5554)
      at java.lang.reflect.Method.invoke(Native Method)
      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:935)
      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:726)
```

**Comment by TrustTheBoy** [comment]
Reason found

**Comment by henninghall** [comment]
Fixed by https://github.com/henninghall/react-native-date-picker/pull/189 and released in v2.7.10
